### DISCUSSION TITLE
Start dev Open data platform

### DISCUSSION THREAD

**Discussion by FWICSS** [opendatateam/udata#3161] [Q&A]
Bonjour,

Je voudrais savoir si l'on peut démarrer un projet a partir du répertoire docker-udata, ou c'est juste un répo pour tester udata ?
deuxième question la visualisation des donnée (udata-tabular-preview) n'est plus directement intégré au template de gouvfr ?

**Comment by maudetes** [comment]
Bonjour,
* `docker-udata` est conçu comme un répertoire de test. N'étant pas utilisé en production, il est possible que vous rencontriez des limitations pour vos besoins.

Concernant la visualisation, elle repose sur :
* un backend de traitement des données tabulaires pour mise en base et APIfication (anciennement [csvapi](https://github.com/etalab/csvapi), maintenant [hydra](https://github.com/datagouv/hydra/))
    * hydra est en cours d'industrialisation mais est une stack complète comprenant une DB PostgreSQL, un crawler, une app et des workers
    * csvapi était une approche plus naïve avec mise en base via SQLite mais sans mise à jour automatique
* l'activation de la fonctionnalité via le plugin [udata-tabular-preview](https://github.com/opendatateam/udata-tabular-preview/)
* un front pour requêter et afficher le contenu apéifié (maintenant intégré dans udata-front directement ([ici](https://github.com/datagouv/udata-front/tree/d8a44d2bea9b18a46c71d984fa5ebc7e753e8504/udata_front/theme/gouvfr/datagouv-components/src/components/ResourceAccordion/Preview)))

**Comment by FWICSS** [comment]
D'accord merci pour c'est précision car j'ai ajouté le plugin [udata-tabular-preview](https://github.com/opendatateam/udata-tabular-preview/) À la version de test, je n'ai obtenu la visualisation sur l'interface. 
Est-il possible de faire une version de développement avec Docker ?
Deuxième question, y a-t-il un moyen de modifier quelques éléments dont le logo du site 
Je travaille dans une collectivité nous regardons les possibilité de mettre en place notre plate forme open source open data

**Comment by rv2931** [comment]
Bonjour
Je suis un peu dans la même situation. La discussion m'intéresse
Le dépôt docker-udata n'inclue pas la dernière version et donc les dernières évolutions et le projet udata ne fournit pas de stack docker complète pour le développement
